### DISCUSSION TITLE
Feel free to use Discussions

### DISCUSSION THREAD

**Discussion by pcaspers** [OpenSourceRisk/Engine#267] [Announcements]
Hi all, feel free to use Discussions to ask questions, propose new features or discuss things with the community!

**Comment by Uge123** [comment]
Hi, could anyone please help me build ORE from source in Mac M2? Facing compatibility issues. Are there any easy ways to build within docker? Thanks

**Comment by pcaspers** [comment]
If you post your specific issues, we can help with that. Just open an issue for it, please.
